api/DocumentOrShadowRoot.json

```
@@ -208,10 +208,12 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot/elementFromPoint",
           "support": {
             "chrome": {
-              "version_added": "53"
+              "version_added": "53",
+              "notes": "Before Chrome 66, this method returned <code>null</code> when the element was a child of a host node. See Chromium issue <a href='https://crbug.com/'>759947'</a> for details."
```

Link directly to an issue, where possible

```suggestion
              "notes": "Before Chrome 66, this method returned <code>null</code> when the element was a child of a host node. See <a href='https://crbug.com/759947'>issue 759947</a>."
```